ecs/components/AvatarShape.proto

```diff
@@ -6,15 +6,19 @@ import "common/Color3.proto";
 
 message PBAvatarShape {
   string id = 1;
-  optional string name = 2;
-  optional string body_shape = 3;
-  optional Color3 skin_color = 4;
-  optional Color3 hair_color = 5;
-  optional Color3 eye_color = 6;
-  repeated string wearables = 7;
-  optional string expression_trigger_id = 8;
-  optional int64 expression_trigger_timestamp = 9;
-  optional string sticker_trigger_id = 10;
-  optional int64 sticker_trigger_timestamp = 11;
-  optional bool talking = 12;
+  optional string name = 2; // default = NPC
+  optional string body_shape = 3; // default = urn:decentraland:off-chain:base-avatars:BaseFemale
+  optional Color3 skin_color = 4; // default = Color3(R = 0.6f, G = 0.462f, B = 0.356f)
+  optional Color3 hair_color = 5; // default = Color3(R = 0.283f, G = 0.142f, B = 0f)
+  optional Color3 eye_color = 6; // default = Color3(R = 0.6f, G = 0.462f, B = 0.356f)
+  optional string expression_trigger_id = 7;
+  optional int64 expression_trigger_timestamp = 8; // default = timestamp
+  optional bool talking = 9;
+  repeated string wearables = 10; // default = [ "urn:decentraland:off-chain:base-avatars:f_eyes_00",
```

Comment:
```suggestion
  /**
   * default = ["urn:decentraland:off-chain:base-avatars:f_eyes_00", 
   *  "urn:decentraland:off-chain:base-avatars:f_eyebrows_00",
   *  "urn:decentraland:off-chain:base-avatars:f_mouth_00" 
   *  "urn:decentraland:off-chain:base-avatars:standard_hair", 
   *  "urn:decentraland:off-chain:base-avatars:f_simple_yellow_tshirt", 
   *  "urn:decentraland:off-chain:base-avatars:f_brown_trousers", 
   *  "urn:decentraland:off-chain:base-avatars:bun_shoes"]
   */
  repeated string wearables = 10;
```

Reply:
For multiline doc it'll be neccesaryr the `/**` , you can commit the suggestion, it was tested.
Without this, this is ts-proto copies the comments:
![image](https://user-images.githubusercontent.com/8042536/188116536-9c96821e-0254-4cd0-ad55-317bf3959558.png)

Reply:
This needs to be changed on the protocol repository. I will change it there and then I will update this one!